### ISSUE TITLE
Global POST_ID issue has returned in a recent update

### ISSUE THREAD

**Issue by purbeckpixels** [pojome/elementor#2149]
The issue with plugins changing global posts and thus affecting the Page Settings and Return to Dashboard functionality of Elementor ([referenced here](https://github.com/pojome/elementor/issues/2077#issuecomment-318915347)) has returned in a recent point update (not sure which one, I've not used Elementor for a few days - I'm using latest version 1.6.3). Again it's the The Events Calendar Shortcode plugin causing the issue (which hasn't been updated since), if I deactivate it everything's fine again.

**Comment by KingYes** [comment]
Please continue there: https://github.com/pojome/elementor/issues/2029

**Comment by purbeckpixels** [comment]
I really don't think the two issues are related. #2029 is dealing with custom CSS in Elementor Pro and The Events Calendar plugin, while the issue I reported involves The Events Calendar Shortcode plugin and the way it uses global post IDs. I helped [diagnose this global post ID issue](https://github.com/pojome/elementor/issues/2077#issuecomment-318915347) during the RC stage of 1.6 and you fixed it in RC4. 1.6 and 1.6.1 continued to work only for the issue/bug to be reintroduced in 1.6.2. We're on 1.6.4 and there's no sign of a fix. It's getting frustrating that I can't edit Page Settings of pages at the moment and the Go to Dashboard link returns me to entirely the wrong page (see the RC thread for more on this).
